common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_common.yaml

```diff
@@ -989,3 +989,52 @@ sBssidInfo:
   downlink_estimated_mac_data_rate_mbps: uint32_t
   uplink_estimated_mac_data_rate_mbps: uint32_t
   sta_measured_uplink_rssi_dbm_enc: uint8_t
+
+sClientConfig:
+  _type: struct
+  stay_on_initial_radio:
+    _type: int8_t
+    _comment: 1 for true, 0 for false, -1 for "not configured".
```

Comment:
Looks like you use the 1,0,-1 a lot, I'd suggest an enum for this.

Reply:
Tried using an enum, but the yaml doesn't accept an enum with a `-1` value